bin/smartdown

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env ruby
 
+Encoding.default_external = 'UTF-8'
```

Comment:
This doesn't seem to relate to the commit message - could you split it out and give some context?